std_execution.bs

```diff
@@ -2563,6 +2611,100 @@ enum class forward_progress_guarantee {
 
     2. Otherwise, `execution::transfer(execution::just(vs...), s)`.
 
+#### `execution::just_error` <b>[execution.senders.just_error]</b> #### {#spec-execution.senders.just_error}
+
+1. `execution::just_error` is used to create a sender that propagates an error to a connected receiver.
+
+    <pre highlight=c++>
+    template&lt;class T>
+    struct <i>just-error-sender</i> // exposition only
+    {
+      T err_;
+
+      template&lt;template&lt;class...> class Tuple, template&lt;class...> class Variant>
+      using value_types = Variant&lt;>;
+
+      template&lt;template&lt;class...> class Variant>
+      using error_types = Variant&lt;T>;
```

Comment:
Shouldn't this also include `std::exception_ptr` since you are handling a throwing `execution::set_value()` call below by sending `std::current_exception()`?

Reply:
Yes, thanks.